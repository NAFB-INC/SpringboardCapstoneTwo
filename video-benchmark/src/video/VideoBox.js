import React from "react";
import Video from "../video/Video";
//import BenchmarkList from "../video/BenchmarkList";

function VideoBox({user,presID,setTime}) {

    function benchmarkAudienceOnly(){
        if(user==="audience"){
            //return <BenchmarkList presID={presID}/>;
        }
        return null;
    }

    return (
        <div className="VideoBox">
            <Video user={user} presID={presID} setTime={setTime}/>
            {benchmarkAudienceOnly()}
        </div>
    );
}

export default VideoBox;