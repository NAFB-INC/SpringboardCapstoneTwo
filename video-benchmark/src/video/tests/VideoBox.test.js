import React from "react";
import { render } from '@testing-library/react';
import VideoBox from '../VideoBox';

it("renders without crashing", function() {
  render(<VideoBox />);
});

it("matches snapshot", function() {
  const { asFragment } = render(<VideoBox />);
  expect(asFragment()).toMatchSnapshot();
});