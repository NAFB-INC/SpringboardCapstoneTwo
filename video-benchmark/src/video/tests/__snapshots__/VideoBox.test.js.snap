// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="VideoBox"
  >
    <div
      class="Video"
    />
  </div>
</DocumentFragment>
`;
